autopush/endpoint.py

```diff
@@ -342,9 +342,12 @@ def _store_auth(self, jwt, crypto_key, token, result):
     def _invalid_auth(self, fail):
         if isinstance(fail.value, VapidAuthException):
             raise fail.value
+        if isinstance(fail.value, JOSEError):
```

Comment:
Yeah, didn't know about the tuple thing. Fixing up the other isinstance set to do the same.